jss.spec

```diff
@@ -324,7 +324,7 @@ touch %{_vpath_builddir}/.targets/finished_generate_javadocs
     --lib-dir=%{_libdir} \
     --sysconf-dir=%{_sysconfdir} \
     --share-dir=%{_datadir} \
-    --cmake=%{__cmake} \
+    --cmake="%{__cmake} -DENABLE_NSS_VERSION_PQC_DEF=ON" \
```

Comment:
This option is only needed on CentOS/RHEL 10, but the CI is running on Fedora 42 so it's failing. We need to use a conditional like this:
```
%if 0%{?rhel} && 0%{?rhel} >= 10
... specify the option ...
%endif
```